Lab3/OnlineMusicPlayer/style.css

```diff
@@ -0,0 +1,122 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;600;700&display=swap');
+*{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
+*::before, *::after{
+  padding: 0;
+  margin: 0;
+}
+:root{
+  --violet: #9f6ea3;
+  --lightblack: #515C6F;
+  --white: #ffffff;
+  --darkwhite: #cecaca;
+  --pinkshadow: #ffcbdd;
+  --black: #000;
+}
+:focus{
+    outline: 1vh solid red;
+}
+
+body{
+  display: flex;
+  align-items: center;
+  justify-content:center;
+  min-width: 10vh;
+  max-height: 50vh;
+  min-width: 50vh;
+  margin-top: 10vh;
+}
+.wrapper{
+  width: 50vh;
+  padding: 0.5vw 0.6vw;
+  overflow: hidden;
+  position: relative;
+  border-radius: 0.5vw;
+  background: var(--white);
+  box-shadow: 0vw 0.5vw 0.5vw var(--black);
+}
+.wrapper i{
+  cursor: pointer;
+  justify-content: center;
+}
+
+.song-details{
+    text-align: center;
+    margin: 2vw 0;
+  }
+  .song-details p{
+    color: var(--lightblack);
+  }
+  .song-details .name{
+    font-size: 0.7vw;
```

Comment:
don't use viewport values.

Reply:
yes.

Reply:
if I requested changes from u, then maybe it means that u need to fix it, innit?

Reply:
Fix.